pyanaconda/ui/gui/spokes/storage.py

```diff
@@ -1019,15 +1030,17 @@ def on_back_clicked(self, button):
         # 3) we are just asked to do autopart => check free space and see if we need
         #                                        user to do anything more
         self._auto_part_enabled = (
-            self._get_selected_partitioning_method() == PARTITIONING_METHOD_AUTOMATIC
+            self._selected_partitioning_method == PARTITIONING_METHOD_AUTOMATIC
         )
         disks = filter_disks_by_names(self._available_disks, self._selected_disks)
         dialog = None
         if not self._auto_part_enabled:
-            if self._get_selected_partitioning_method() == PARTITIONING_METHOD_CUSTOM:
+            if self._selected_partitioning_method == PARTITIONING_METHOD_CUSTOM:
                 self.skipTo = "CustomPartitioningSpoke"
-            if self._get_selected_partitioning_method() == PARTITIONING_METHOD_BLIVET:
+            if self._selected_partitioning_method == PARTITIONING_METHOD_BLIVET:
                 self.skipTo = "BlivetGuiSpoke"
+            if self._selected_partitioning_method == PARTITIONING_METHOD_UNKNOWN_MANUAL:
+                log.log("UNKNOWN partitioning method made it too far!")
```

Comment:
Shouldn't this be an error? Is there a valid scenario for back button clicked and no partitioning method chosen?

Reply:
Probably yes.